components/conversation/src/Conversation.svelte

```diff
@@ -522,17 +476,6 @@
                     {:else}
                       <p class="snippet">{message.snippet}</p>
                     {/if}
-                    {#if !feedbackSubmittedMessages.includes(message.id)}
```

Comment:
Decided to nix the Clean Conversation feedback button; it was originally for internal testing only.